streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedTable.java

```diff
@@ -35,6 +38,8 @@
      * @param adder         the instance of {@link Reducer} for addition
      * @param subtractor    the instance of {@link Reducer} for subtraction
      * @param name          the name of the resulted {@link KTable}
+     * @return a {@link KTable} with the same key and value types as this {@link KGroupedTable}
```

Comment:
I like "containing" better than "that contains" in this context. Not sure about the comma thing... Where are the native speakers?